Source/WebInspectorUI/UserInterface/Views/DetailsSection.js

```diff
@@ -50,15 +50,16 @@ WI.DetailsSection = class DetailsSection extends WI.Object
             this._headerElement.appendChild(this._optionsElement);
         }
 
-        this._titleElement = document.createElement("span");
-        this._headerElement.appendChild(this._titleElement);
+        if (title)
+            this.title = title;
+        else
+            this._headerElement.setAttribute("inert", true);
```

Comment:
What about a header with options but no title?
Also see my comment about just applying the `hidden` class instead if we don't need the element.

Reply:
Using the `hidden` class is a great idea. It simplified a lot. Thank you!

> What about a header with options but no title?

With the current changes, it would be hidden until the implementation is adjusted.

There is no current use case for a `DetailsSection` with a collapsible header _without_ a title but _with_ an options element. It would be a strange UI. Whoever ends up needing it will have to adjust toggling the `hidden` class accordingly.

Reply:
Replaced use of `hidden` CSS class with `hidden` element attribute.